src/ocean/core/VersionCheck.d

```diff
@@ -0,0 +1,28 @@
+module ocean.core.VersionCheck;
+
+import ocean.transition;
+import ocean.core.Traits;
+
+public template hasFeaturesFrom ( istring libname, ulong major, ulong minor )
+{
+    const hasFeaturesFrom = hasFeaturesFromFunc!(libname, major, minor)();
+}
+
+private bool hasFeaturesFromFunc ( istring libname, ulong major, ulong minor ) ()
+{
+    struct Library
+    {
+        mixin("import " ~ libname ~ ".LibVersion;");
+    }
+
+    mixin("return is(typeof(Library.has_features_"
+        ~ ctfe_i2a(major) ~ "_" ~ ctfe_i2a(minor) ~ "));");
```

Comment:
So in this case, how do we deprecate `has_features` symbols when a library is using `-de` ?